command/harness/destroy.go

```diff
@@ -29,6 +29,7 @@ type VMCleanupRequest struct {
 	StageRuntimeID string  `json:"stage_runtime_id"`
 	LogKey         string  `json:"log_key,omitempty"`
 	Context        Context `json:"context,omitempty"`
+	TaskID         string  `json:"task_id,omitempty"`
```

Comment:
Would be better to add this to `Context`. It will automatically start appearing in the logs if it's available. Check `AddContext` function.

Reply:
Good point. Done.